gio/Gir.toml

```diff
@@ -1565,6 +1565,92 @@ concurrency = "send+sync"
 cfg_condition = "unix"
     [[object.derive]]
     name = "Debug"
+    [[object.function]]
+    name = "compare"
```

Comment:
As this file is already too huge, I would suggest using pattern instead of name to combine those similar configs

Reply:
Putting them all into the same line with `|`?

Reply:
yep, that would be easier & can be documented as well